css/custom.css

```diff
@@ -169,10 +169,6 @@ nav{
   -webkit-transition: top 0.7s ease-in-out !important;
   transition: top 0.7s ease-in-out !important;
 }
-.mobile-menu-toggle{
-  position: absolute;
-  top: -7px;
```

Comment:
Can you put it inside a viewpoint? Now the desktop site displays the toogle button.

Reply:
The desktop site already displaying the toggle button. The issue is "It is not showing in mobile view". 
Do you want me to remove toggle in desktop site display?